components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/commons/lib/ace-editor/mode/asyncapi.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define("ace/mode/asyncapi", ["require", "exports", "module", "ace/lib/oop", "ace/mode/text", "ace/mode/asyncapi_highlight_rules", "ace/range", "ace/mode/folding/asyncapi"],
+    function (require, exports, module) {
```

Comment:
this only has three modules but your define has more stuff

Reply:
fixed with https://github.com/wso2/carbon-analytics/pull/1913/commits/d807fbc380a7203f1238bcad5425d84778a91878